### ISSUE TITLE
Workshop issue: Che 2.0.44 didn't start

### ISSUE THREAD

**Issue by rafabene** [openshiftio/openshift.io#1255]
When I open the Che workspace, I can see only HTTP Status 403 -

**Comment by ibuziuk** [comment]
@rafabene could you share che-server stack trace ?

**Comment by ibuziuk** [comment]
was able to reproduce:


```
2017-10-31 17:59:38,791[nio-8080-exec-4]  [ERROR] [c.r.che.valve.UserAuthValve 134]     - Exception while validating user
--
  | java.io.IOException: Server returned HTTP response code: 400 for URL: http://localhost:8080/api/token/user
  | at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1876)
  | at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1474)
  | at com.redhat.che.valve.UserAuthValve.userMatches(UserAuthValve.java:124)
  | at com.redhat.che.valve.UserAuthValve.authenticate(UserAuthValve.java:80)
  | at org.keycloak.adapters.tomcat.KeycloakAuthenticatorValve.doAuthenticate(KeycloakAuthenticatorValve.java:54)
  | at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:556)
  | at org.keycloak.adapters.tomcat.AbstractKeycloakAuthenticatorValve.invoke(AbstractKeycloakAuthenticatorValve.java:187)
  | at com.redhat.che.valve.UserAuthValve.invoke(UserAuthValve.java:103)
  | at org.apache.catalina.valves.rewrite.RewriteValve.invoke(RewriteValve.java:580)
  | at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140)
  | at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
  | at ch.qos.logback.access.tomcat.LogbackValve.invoke(LogbackValve.java:256)
  | at org.apache.catalina.valves.RemoteIpValve.invoke(RemoteIpValve.java:677)
  | at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
  | at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:349)
  | at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:783)
  | at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
  | at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:798)
  | at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1434)
  | at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
  | at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  | at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  | at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
  | at java.lang.Thread.run(Thread.java:748)
  | 2017-10-31 17:59:39,116[nio-8080-exec-7]  [ERROR] [c.r.che.valve.UserAuthValve 134]     - Exception while validating user
  | java.io.IOException: Server returned HTTP response code: 400 for URL: http://localhost:8080/api/token/user
  | at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1876)
  | at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1474)
  | at com.redhat.che.valve.UserAuthValve.userMatches(UserAuthValve.java:124)
  | at com.redhat.che.valve.UserAuthValve.authenticate(UserAuthValve.java:80)
  | at org.keycloak.adapters.tomcat.KeycloakAuthenticatorValve.doAuthenticate(KeycloakAuthenticatorValve.java:54)
  | at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:556)
  | at org.keycloak.adapters.tomcat.AbstractKeycloakAuthenticatorValve.invoke(AbstractKeycloakAuthenticatorValve.java:187)
  | at com.redhat.che.valve.UserAuthValve.invoke(UserAuthValve.java:103)
  | at org.apache.catalina.valves.rewrite.RewriteValve.invoke(RewriteValve.java:580)
  | at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140)
  | at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
  | at ch.qos.logback.access.tomcat.LogbackValve.invoke(LogbackValve.java:256)
  | at org.apache.catalina.valves.RemoteIpValve.invoke(RemoteIpValve.java:677)
  | at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
  | at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:349)
  | at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:783)
  | at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
  | at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:798)
  | at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1434)
  | at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
  | at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  | at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  | at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
  | at java.lang.Thread.run(Thread.java:748)
```

Looks like che  2.0.44 is entirely broken. Working on hot-fix

**Comment by ibuziuk** [comment]
Fixed in `2.0.45`
